.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -18,18 +18,18 @@ assignees: ''
 <!-- If applicable, specify the package manager version you're using (e.g., npm 7.1, pip-compile 5.0, etc.) -->
 **Language version**
 <!-- If applicable, specify the language version you're using (e.g., node 14.1, Ruby 2.7, etc. ) -->
-**Manifest location and content prior to update**
+**Manifest location and content before update**
```

Comment:
```suggestion
**Manifest location and content before the Dependabot update**
```

👍🏻  on the before/prior to change - 'the update' seems ambiguous when I read it back so let's be specific.